snow/networking/router/subnet_router.go

```diff
@@ -21,15 +23,19 @@ type ChainRouter struct {
 	lock     sync.RWMutex
 	chains   map[[32]byte]*handler.Handler
 	timeouts *timeout.Manager
+	gossiper *timer.Repeater
 }
 
 // Initialize the router
 // When this router receives an incoming message, it cancels the timeout in [timeouts]
 // associated with the request that caused the incoming message, if applicable
-func (sr *ChainRouter) Initialize(log logging.Logger, timeouts *timeout.Manager) {
+func (sr *ChainRouter) Initialize(log logging.Logger, timeouts *timeout.Manager, gossipFrequency time.Duration) {
```

Comment:
Can you specify what `gossipFrequency` is in the method spec?

Reply:
Added to the docs